Source/WebKit/Platform/IPC/StreamServerConnection.cpp

```diff
@@ -128,7 +128,7 @@ void StreamServerConnection::didClose(Connection&)
     // Client is expected to listen to didClose from the main connection.
 }
 
-void StreamServerConnection::didReceiveInvalidMessage(Connection&, MessageName, int32_t)
+void StreamServerConnection::didReceiveInvalidMessage(Connection&, MessageName name, int32_t index)
```

Comment:
Why this change?

Reply:
Ah, good catch, I had some debug code here, and I forgot to remove the parameter names again :) Will fix!